src/components/AddFacilityModal/AddFacilityModal.jsx

```diff
@@ -30,6 +30,7 @@
 const AddFacilityModal = () => {
   const { isOpen, onOpen, onClose } = useDisclosure();
   const [scrollBehavior] = React.useState('inside');
+  const [facilityToDelete, setFacilityToDelete] = useState(null);
```

Comment:
🚫 **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'setFacilityToDelete' is assigned a value but never used.